mythril/laser/ethereum/instructions.py

```diff
@@ -1057,9 +1057,45 @@ def returndatacopy_(self, global_state: GlobalState) -> List[GlobalState]:
         :param global_state:
         :return:
         """
-        # FIXME: not implemented
         state = global_state.mstate
-        start, s2, size = state.stack.pop(), state.stack.pop(), state.stack.pop()
+        memory_offset, return_offset, size = (
+            state.stack.pop(),
+            state.stack.pop(),
+            state.stack.pop(),
+        )
+        concrete_size = None
+        concrete_return_offset = None
+
+        try:
+            concrete_memory_offset = helper.get_concrete_int(memory_offset)
+        except TypeError:
+            log.debug("Unsupported symbolic memory offset in RETURNDATACOPY")
+            return [global_state]
+
+        try:
+            concrete_return_offset = helper.get_concrete_int(return_offset)
+        except TypeError:
+            log.debug("Unsupported symbolic return offset in RETURNDATACOPY")
```

Comment:
If we return [globalstate] for these catches, then we don't need the ifstatement at 1090. Additionally we won't need ln 1066, 1067 if we do that, as everything would always be initialized after the try catches

Reply:
Why did you resolve this?

Reply:
Because I added `return [global_state]` to these catches and fixed lines 1090, 1066, and 1067.